reactivesearch/src/ts/OpenLogClient.ts

```diff
@@ -0,0 +1,169 @@
+import axios, {AxiosRequestConfig, AxiosProxyConfig, AxiosInstance} from "axios";
+/**
+ * Methods and client to talk directly with OpenSearch for logging
+ * 
+ */
+
+
+
+/**
+ * Class to handle OpenSearch authentication (eventually) log connectivity
+ */
+export class OpenLogClient {
```

Comment:
We should open a seperate ticket to potentitally get these classes either added to SearchCollector project or be their own Github repo....